x-pack/plugins/observability_solution/infra/public/components/asset_details/tabs/overview/services.tsx

```diff
@@ -44,13 +45,19 @@ export const ServicesContent = ({
     app: 'apm',
     pathname: '/onboarding',
   });
+
+  const parsedDateRange = useTimeRange({
+    rangeFrom: dateRange.from,
+    rangeTo: dateRange.to,
+  });
+
   const params = useMemo(
     () => ({
       filters: { [HOST_NAME_FIELD]: hostName },
-      from: dateRange.from,
-      to: dateRange.to,
+      from: parsedDateRange.from,
```

Comment:
Ideally, yes, because it's a dependency of `useFetcher` and the `useFetcher` dependency array acts the same way the `useEffect` does, meaning everything in there should preferably be memoized.